Assets/Scripts/Combat/Melee/CollisionDetection.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CollisionDetection : MonoBehaviour
+{
+    public SwordController sc;
+    public GameObject HitParticle;
+
+    void OnTriggerEnter(Collider other)
+    { 
+        if(other.gameObject.tag == "QuestObject" || other.gameObject.tag == "Enemy") {
```

Comment:
We moeten wat met deze tags, ik neem namelijk aan dat een quest objective niet per se een enemy is maar ook een object wat je moet meenemen ofzo

Reply:
Heb t iig zo overgenomen van de FiringGun class... Ik wilde het eerst zo ff laten werken, en dan daarna mogelijk aanpassen bij beide. Dit werkte iig wel voor playtest 1 met shooting

Reply:
Playtest 1 was questobjective ook enemy's. Hoeft niet bij playtest 2 maar dan kijken we dan